bin/hdfcoinc/pycbc_page_segtable

```diff
@@ -108,8 +108,11 @@ for segment_name in opts.segment_names:
 
     # put comment in caption
     caption += " " + segment_name
-    if comment_dict[key] != None:
-        caption += " ("+comment_dict[key]+")"
```

Comment:
Well it works. But no one ever put comments in the segments XML files generated by pycbc so we don't see anything. If you want to remove it fine.